src/libs/ReportUtils.js

```diff
@@ -874,8 +874,14 @@ function getIconsForParticipants(participants, personalDetails) {
         ]);
     }
 
-    // Sort all logins by first name (which is the second element in the array)
-    const sortedParticipantDetails = participantDetails.sort((a, b) => a[2] - b[2]);
+    // Sort all logins by first name (position 2 element in array)
+    // if multiple participants have the same first name, sub-sort them by login/displayName (position 1 element in array)
+    // if that still clashes, sub-sort by accountID (position 0 element in array)
+    const sortedParticipantDetails = _.chain(participantDetails)
```

Comment:
@honnamkuan Can you confirm if chaining the sortBy won't have side effects?  Also quick question, would it make sense to use this instead?

```
const sortedParticipantDetails = _.orderBy(participantDetails, [2, 1, 0]);
```

let me know your thoughts.

Reply:
@honnamkuan Expensify App uses [lodash](https://github.com/Expensify/App/blob/c435734d8cf360e964e20eea1fb54c541f6fa7ea/package.json#L94).

Reply:
No. Thanks this works. I just went through the SO thread. I will test this shortly.